README.md

```diff
@@ -117,6 +117,17 @@ const getUserInfoList = async () => {
 ## Common Customization Scenarios
 Feel free to fork this repository and make your own modifications to the UX or backend logic. For example, you may want to change aspects of the chat display, or expose some of the settings in `app.py` in the UI for users to try out different behaviors. 
 
+### Scalability
+For apps published with `az webapp up` or from the Azure AI Studio, you can increase your app's ability to handle concurrent requests from multiple users with the following steps:
```

Comment:
FYI, you could also add a gunicorn.conf.py in the App src itself to dynamically compute the optimal number of workers, it's what we often do for our App Service apps, i.e.

https://github.com/Azure-Samples/azure-django-postgres-flexible-appservice/blob/246b1e3ad98b3c11f74682a4f82e9a18d3b58911/src/gunicorn.conf.py#L7

Reply:
Thank you, I will look into this!